settings.py

```diff
@@ -782,5 +782,5 @@ def _theme_completions(view):
                 "{}  \ttheme".format(theme),
                 theme
             )
-            completions.add(item)
+            completions.append(item)
```

Comment:
Changing this to a list without re-adding the `if item not in completions` check will result in duplicates in the results.
Sets are the proper structure to use here (and imo severely underused in many projects) and a list should be returned at the end of the function (see comment above).